Client/src/Pages/Maintenance/CreateMaintenanceWindow/index.jsx

```diff
@@ -66,50 +77,80 @@ const CreateNewMaintenanceWindow = () => {
     unit: "minutes",
     displayName: "",
     AddMonitors: "",
+    autoCompleteValue: {},
   });
   const [errors, setErrors] = useState({});
 
   const handleChange = (event, name) => {
-    const { value } = event.target;
-    setValues((prev) => ({
-      ...prev,
-      [name]: value,
-    }));
+    if (event.target === undefined) {
+      setValues((prev) => ({
+        ...prev,
+        autoCompleteValue: {
+          _id: event._id,
+          name: event.name,
+        },
+      }));
+    } else {
+      const { value } = event.target;
+      setValues((prev) => ({
+        ...prev,
+        [name]: value,
+      }));
+    }
+  };
+
+  const convertDurationToMilliseconds = (duration, unit) => {
+    let milliseconds = 0;
+    if (unit === "minutes") {
+      milliseconds = duration * 60 * 1000;
+    } else if (unit === "hours") {
+      milliseconds = duration * 60 * 60 * 1000;
+    } else if (unit === "days") {
+      milliseconds = duration * 24 * 60 * 60 * 1000;
```

Comment:
We should avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) in our code.

These can be refactored as
```
MS_PER_MINUTE = 60000
MS_PER_HOUR = 360000
MS_PER_DAY = 86400000
```
for clarity